docs/docs/examples/repl.md

```diff
@@ -34,43 +34,57 @@ Steps
 
         opa run
 
-    Without any data, you can experiment with simple boolean expressions to get the hang of it:
-
-        > a = 1, b = 2, a != b
-        +---+---+
-        | A | B |
-        +---+---+
-        | 1 | 2 |
-        +---+---+
-        > a = 1, b = 2, a = b
+    Without any data, you can experiment with simple expressions to get the hang of it:
+
+        > true
+        true
+        > 3.14
+        3.14
+        > ["hello", "world"]
+        [
+          "hello",
+          "world"
+        ]
+
+    You can also test simple boolean expressions:
+
+        > true = false
         false
-        > 10 > 9
+        > 3.14 > 3
         true
-        > a = [1,2,3,4], a[i] > 2
-        +-----------+---+
-        |     A     | I |
-        +-----------+---+
-        | [1,2,3,4] | 2 |
-        | [1,2,3,4] | 3 |
-        +-----------+---+
+        > "hello" != "goodbye"
+
+    When you enter expressions into the REPL, you are effectively running *queries* against OPA. The REPL output shows the values of variables in the expression that make the query **true**. If there is no set of variables that would make the query true, the REPL prints **false**. If there are no variables in the query and the query evaluates successfully, then the REPL just prints **true**.
```

Comment:
I would think we'd want to move this til after the pi > 3 stuff that people know and start with something like...

The powerful thing about the REPL is that you can run queries using the same syntax that you would use to lookup values.  For example if i has value 0 then data.servers[i] returns the first value in the data.servers array.  That same expression data.servers[i] when i has no value defines a query that returns all the values of i and data.servers[i].  The REPL displays the results of the query as a table.